v2/package.json

```
@@ -75,7 +74,7 @@
   },
   "main": "lib/index.js",
   "license": "Apache-2.0",
-  "version": "0.0.0",
+  "version": "1.9.0",
```

autogenerated on release
```suggestion
  "version": "0.0.0",
```